flycheck.el

```diff
@@ -10668,6 +10668,18 @@ When non-nil, pass the directory via the `--workdir' option."
   :package-version '(flycheck . "32"))
 (make-variable-buffer-local 'flycheck-ghdl-workdir)
 
+(flycheck-def-option-var flycheck-ghdl-ieee-library nil vhdl-ghdl
+  "The standard to use for the ieee library
```

Comment:
First sentence of a docstring always ends with a colon.

Reply:
Do you mean a period? I don't see a colon at the end of the other docstrings. I also noticed the docstring for the option above my addition (flycheck-ghdl-workdir) is missing a period. Should I add that or make a separate PR,  since it doesn't have to do with the option I'm adding?

Reply:
Yes of course, period.  My bad.

Go ahead and fix the other docstring.  No need for a separate PR :)

Reply:
Fixed both docstrings in latest commits.